azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/LocalizationConfiguration.kt

```diff
@@ -0,0 +1,18 @@
+package com.azure.android.communication.ui.configuration
+
+class LocalizationConfiguration {
```

Comment:
Shauna, please check with Pavel, I think all public classes are in Java? (please confirm)

Reply:
@ShaunaSong, @iaulakh - yes, all public API should be in Java